src/contracts/interfaces/IRegistryCoordinator.sol

```diff
@@ -30,8 +30,6 @@ interface IRegistryCoordinator {
     struct Operator {
         // the id of the operator, which is likely the keccak256 hash of the operator's public key if using BLSRegsitry
         bytes32 operatorId;
-        // start taskNumber from which the  operator has been registered
-        uint32 fromTaskNumber; // TODO: REMOVE
```

Comment:
did we decide this isn't useful at all?
IIRC the main use was for payments, which I guess we can rely on the IndexRegistry (or another sub-registry) for, if needed?

Reply:
yes this can likely be taken out, I'll do it in another PR

Reply:
> yes this can likely be taken out, I'll do it in another PR

We should realistically have a design meeting to focus on some issues like this and try to sort out clearer requirements. I like what you've done but want to try to get ahead of future design problems that will cause us to do a lot of revisions later.
I'm OK with this for now, but let's work on putting something together for a discussion?